README.md

```diff
@@ -19,13 +19,43 @@ In order to render and preview the site locally (without docker) you will need a
 
 ## "Publishing" your changes
 
-Publishing is now done by the jenkins pipeline. Once a PR is merged to master, the changes will be reflected on the documentation site. 
+Publishing is done by the jenkins pipeline. Once a PR is merged, the changes will be reflected on the documentation site, hosted under [gh-pages] branch and served by Github Pages.
+
+The different versions of the documentation are maintained in separate branches.
+The `main` branch hosts the source files for the version that is under development as well as the following **production site files**:
+- `docs/CNAME` - DNS record which tells Github Pages to serve the content at https://docs.edgexfoundry.org instead of https://edgexfoundry.github.io/edgex-docs
+- `docs/index.html` - site index page that redirects from `/` to `/{latest-release}`
+- `docs/versions.json` - version info to populate the site version drop-down menu
+- `versions.json` - identical to `docs/versions.json`. Unknown purpose!
```

Comment:
@ernestojeda is the last files used by the Jenkins pipeline? If not, we can remove it.

Reply:
Jenkins does not use any of the files specifically that I know of. It just calls `mkdocs` and copies the generated files in the docs folder into the `gh-pages` branch.

Reply:
Okay. Since this is the case, I went ahead and removed that file.